allocations/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+import json
+import logging
+from django.conf import settings
+
+logger = logging.getLogger('allocations')
+
+class Allocation(models.Model):
```

Comment:
can you check-in the migration generated by this model, should be something like `allocations/migrations/0001_initial.py`, migrations can get tangled up otherwise, checking them in helps to make sure everyone's on the same page